Makefile

```diff
@@ -1,3 +1,16 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
Same here with the Makefile

Reply:
There are boilerplate templates for `Makefile` and the python script, copied from capa (which was copied from another kubernetes repo), enforces this. Also, capa `Makefile` has the boilerplate stuff.